source/documents/modify-multiple.txt

```diff
@@ -14,4 +14,151 @@ Modify Multiple Documents
    :depth: 1
    :class: singlecol
 
-You can perform bulk update operations on multiple documents in Compass.
\ No newline at end of file
+You can perform bulk update operations on multiple documents in Compass
+by using the :guilabel:`Update Documents` modal. Performing updates with
+the :guilabel:`Update Documents` modal helps you visualize updates 
+to your data before you apply them.
+
+About this Task
+---------------
+
+- Any syntax that works with the ``update`` parameter of the
+  :method:`db.collection.updateMany()` is valid. 
+  
+- The :guilabel:`Update Documents` modal does not support any ``options`` 
+  parameters such as upsert, writeConcern, or collation.
+
+Before you Begin
+----------------
+
+The :guilabel:`Update Documents` modal is available starting
+in Compass version ``1.5.X``. For instructions on updating or installing
```

Comment:
Version is not yet confirmed, will update this when the exact release version is known.